packages/documentation-site/patternfly-docs/content/design-guidelines/patterns/actions/actions.md

```diff
@@ -0,0 +1,91 @@
+1. [**Action icon (button):**](/components/button) Icon buttons are commonly used to trigger recognizable and recurring actions. For example, ellipsis icons are used to open menus.
+1. [**Action menu item:**](/components/menus/menu/design-guidelines#action-menu) To save space, actions can be placed within a menu. This is also useful when there are multiple actions that a user can take. 
+
+## Usage
+
+Use actions to make it clear and convenient for users to complete tasks in a UI. When applied intentionally, actions can make the UI more efficient and effective. 
+
+When labeling or referring to actions in your UI, make sure that you adhere to our [terminology guidelines](/ux-writing/terminology) and align any icons you use with the usage behavior outlined in our [icon design foundations.](/design-foundations/icons#all-icons) 
+
+### Deletion vs removal 
+Make sure that users understand the purpose and consequences of deleting or removing data objects, items, or similar resources via the UI. Deleting is different from removing. 
+
+- **Deletion:** Refers to the destruction of information (for example, deleting a group, system, or a role.) Often identified by a trash icon.
+
+- **Removal:** Refers to removing the content/system from a particular view, without the destruction of information (for example, removing something from a group or removing a user.) Often identified by a solid minus-circle icon.
+
+#### Deletion 
+
+When resource deletion is irreversible, there is a high risk that users will lose highly important information or data, so you should ensure that they understand the impact of deletion. 
+
+Users can typically trigger deletion using buttons or menu items. Consider the following for deletion triggers:
+- Button triggers should use secondary button styling. 
+- Menu item triggers (like within an options/kebab menu) should use default text styling. 
+- Do not use red danger styling for deletion buttons or menu items, because they will both launch confirmation modals before completing the deletion. 
+
+After a user triggers deletion, communicate that it will be permanent via a confirmation modal:
+
+![Confirmation modal with danger styling.](./img/critical-deletion-modal.png)
+
+1. A **warning icon** beside the title.
+1. A **labeled text input** to ask users to confirm the resource name. 
+    - An error will be shown if the user does not enter a resource name, or if the resource name does not match the resource being deleted.
+1. A **danger button** that submits the modal.
+
+Once a resource has been successfully deleted, show an [inline success alert:](/components/alert#inline-alerts-variants)
+
+![Inline success alert for deletion.](./img/critical-deletion-success.png)
+
+#### Removal
+When the action of removing a resource can be reversed, it has a lower impact on the user experience. A confirmation dialog isn't necessary because users can easily retrieve the deleted resource. 
```

Comment:
there's another "delete" later in the paragraph to edit 😀

Reply:
oml 😅 let me push that too. Thanks for bearing with me!